docs/luigi-compound-container-api.md

```diff
@@ -38,6 +46,14 @@ Type: [Object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Globa
 **Meta**
 
 *   **since**: 1.0.0
+TODO: better example
+```js
+myContainer.compoundConfig = {foo: bar}
+```
+
+```HTML
+<luigi-container viewUrl="/index.html" compoundConfig='{foo: bar}'></luigi-container>
```

Comment:
Not sure if we should provide an example for compould attribute in html

Reply:
given the size of it, it seems reasonable to not wanting people to do it this way so I removed it